en/integrations/plugin-packs/procedures/blockchain-parity-restapi.md

```diff
@@ -0,0 +1,165 @@
+---
+id: blockchain-parity-restapi
+title: Parity API
+---
+
+## Overview
+
+Parity Blockchain is a fork of the Ethereum Blockchain. 
+
+The Centreon Plugin Pack *Parity API* aims to get information and metrics from the 
+Parity API. 
+
+## Pack assets
+
+### Monitored objects
+
+* Blockchain Parity nodes
+     * Parity
+     * Info
+     * Eth
+     * Net
+
+### Collected metrics & status
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Parity-->
+
+| Metric name                 | Description                      | Unit |
+|:----------------------------|:---------------------------------|------|
+| parity.pending.transactions | Number of pending transactions   |      |
+| parity.mempol.usage         | Memory pool usage                |   B  |
+| parity.mempol.size          | Memory pool size                 |   B  |
+| parity.peers.connected      | Number of connected peers        |      |
+| parity.peers.max            | Maximum number of peers          |      |
+| parity.peers.usage          | Peers usage expressed in percent |   %  |
+
+<!--Eth-->
+
+| Metric name                          | Description                   |
+|:-------------------------------------|:------------------------------|
+| parity.eth.sync.status               | State of node synchronization |
+| parity.eth.block.gas                 | Gas                           |
+| parity.eth.block.usage               | Block Usage                   |
+| parity.eth.block.size                | Size of Block                 |
+| parity.eth.block.transactions.number | Number of block transactions  |
+| parity.eth.block.uncles              | Number of block uncles        |
+| parity.eth.gas.limit                 | Maximum Gas available         |
+| parity.eth.gas.price                 | Gas price                     |
+| parity.eth.gas.used                  | Gas consumption               |
+
+<!--Net-->
+
+| Metric name                   | Description              |
+|:------------------------------|:-------------------------|
+| parity.network.peers.count    | Number of known peers    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To get data from the API, the Poller must be able to communicate with it over the 
+configured port (default: 8545). 
```

Comment:
No documentation available about "how to enable the API" ?

Reply:
No, it's by default.